appinventor/appengine/src/com/google/appinventor/client/explorer/youngandroid/GalleryList.java

```diff
@@ -427,6 +427,8 @@ private void refreshApps(GalleryAppListResult appsResult, int requestId, boolean
         }else{
           appSearchTab.getNoResultsFound().setVisible(false);
         }
+        if (refreshable){appSearchCounter = 0;}
```

Comment:
@toropeza 

Can you reformat this as

if (refreshable) {
   appSearchCounter = 0;
}

(using the correct indentation level)